pkg/registry/service/rest_test.go

```diff
@@ -689,7 +689,7 @@ func TestServiceRegistryIPUpdate(t *testing.T) {
 
 	update = deepCloneService(created_service)
 	update.Spec.Ports[0].Port = 6503
-	update.Spec.ClusterIP = "1.2.3.76" // error
+	update.Spec.ClusterIP = "1.2.4.76" // Error: Provided Cluster IP is not in the range of 1.2.3.0/24
```

Comment:
I think the idea behind this test was to check whether it is possible to change the ClusterIP (it shouldn't be - see:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/validation/validation.go#L1181

This test works fine for me in head (maybe there are some platform/system differences cause it to fail for you).
I think we should try to debug it a bit deeper what's happening there.
Can you please (locally):
1. revert your change
2. add some logging to see what is the value of "updated_service" (sth like glog.Infof("%#v", updated_service)
3. rerun the test the same way that it was failing for you and post the result here?